app/pages/projects.cjsx

```diff
@@ -9,7 +9,9 @@ ProjectCard = require '../partials/project-card'
 
 counterpart.registerTranslations 'en',
   projectsPage:
-    title: 'Projects'
+    title: 'All Projects'
+    showing: 'Showing'
+    found: 'found'
```

Comment:
For the future: the translator can interpolate strings like `showing: 'Showing %(count)s found'`. Probably easier to translate later.